webpack.config.js

```diff
@@ -11,6 +12,10 @@ const baseConfig = {
   module: {
     loaders: [
       {
+        test: /\.css$/,
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'),
```

Comment:
Ok cool - let's leave it as is for now - this should allow us to carry on with the user story while developing locally. We can figure out the rest of the deploy process once everyone are back.